packages/features/ee/video/ViewRecordingsDialog.tsx

```diff
@@ -0,0 +1,159 @@
+import { useSession } from "next-auth/react";
+import { useState } from "react";
+
+import dayjs from "@calcom/dayjs";
+import LicenseRequired from "@calcom/features/ee/common/components/v2/LicenseRequired";
+import { WEBSITE_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { RouterOutputs, trpc } from "@calcom/trpc/react";
+import type { PartialReference } from "@calcom/types/EventManager";
+import type { RecordingObj } from "@calcom/types/VideoApiAdapter";
+import { Dialog, DialogClose, DialogContent, DialogFooter, DialogHeader } from "@calcom/ui";
+import { Loader, Button, showToast, Icon } from "@calcom/ui";
+
+import TeamsUpgradeRecordingBanner from "./components/TeamUgradeRecordingBanner";
+
+type BookingItem = RouterOutputs["viewer"]["bookings"]["get"]["bookings"][number];
+
+interface IViewRecordingsDialog {
+  booking?: BookingItem;
+  isOpenDialog: boolean;
+  setIsOpenDialog: React.Dispatch<React.SetStateAction<boolean>>;
+  timeFormat: number | null;
+}
+
+function convertStoMs(seconds: number) {
```

Comment:
```suggestion
function convertSecondsToMs(seconds: number) {
```
readability over shortness